packages/notifi-react-card/lib/components/defaults.css

```diff
@@ -879,6 +883,10 @@ input::-webkit-inner-spin-button {
   overflow-wrap: anywhere;
 }
 
+.NotifiUserInfoPanel__email {
```

Comment:
is this only necessary for email? what about Telegram or SMS?

Reply:
yeah, I think we should add gap on all the destinations